Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -484,6 +485,22 @@ static ExceptionOr<void> forEachRendererInPaintOrder(NOESCAPE const std::functio
     return { };
 };
 
+static bool rendererIsFragmented(const RenderLayerModelObject& renderer)
+{
+    if (renderer.hasContinuationChainNode() || renderer.isContinuation())
+        return true;
+
+    CheckedPtr box = dynamicDowncast<RenderBox>(renderer);
+    if (!box)
+        return false;
+
+    CheckedPtr enclosingFragmentedFlow = renderer.enclosingFragmentedFlow();
```

Comment:
Do we care about fragmented inline boxes?

Reply:
The VT spec explicitly mentions splitting inlines across line boxes as a case that should not prevent view transitions.